python_modules/dagster/setup.py

```diff
@@ -58,7 +58,9 @@ def get_version() -> str:
             # alembic 1.7.0 is a breaking change
             "alembic>=1.2.1,!=1.6.3,<1.7.0",
             "croniter>=0.3.34",
-            "grpcio>=1.32.0",  # ensure version we require is >= that with which we generated the grpc code (set in dev-requirements)
+            # grpcio 1.48.1 has hanging/crashing issues: https://github.com/grpc/grpc/issues/30843
+            # ensure version we require is >= that with which we generated the grpc code (set in dev-requirements)
+            "grpcio>=1.32.0,<1.48.1",
```

Comment:
maybe `!=` instead of `<` but its an issue again so maybe safest. Somehow need to remember to drop these.